### DISCUSSION TITLE
extend search function

### DISCUSSION THREAD

**Discussion by gitchris8** [benjaminjonard/koillection#461] [Ideas]
Good day,
is it possible to extend the search function? I want to search for fields that I have created in the templates.
As an example for IT devices: search by serial number

Thanks a lot
Chris

**Comment by benjaminjonard** [comment]
Hello,

For now as an easy and quick way to improve the search function, I updated the search engine to take into account items fields (restricted to text fields for this first version).
If you type the serial number in the search field, you should see your item

**Comment by gitchris8** [comment]
Thanks for the quick implementation :)

**Comment by codemonky** [comment]
Would it be possible to allow searching when a user profile visibility is set to Public?

**Comment by benjaminjonard** [comment]
Hello, yes I'll try to include this in the next release
